docs/build-nix.md

```diff
@@ -0,0 +1,66 @@
+# Building on Nix
+
+The Nix Ecosystem insists on the purity of its builds, therefore compiling 
+Dosbox Staging on it requires slightly different inputs so meson gets the 
+appropriate environment variables from pkg-config.
+
+## Installing the Dependencies under Nix
+
+Nix has an unique toolset where you don't have to install it's packages to run 
+them. However, if you still want to, instead of using `nix-env` to install your 
+dependencies like your traditional package manager, we recommend you to do the 
+following:
+
+### Install dependencies with Home Manager (Recommended)
+
+1. Install Home-Manager: <https://github.com/nix-community/home-manager#installation>.
+2. After following the guide add the following packages on home.nix:
```

Comment:
@kcgen I feel like this needs to better phrased. Any thoughts?

Reply:
Comma after guide.